compute_endpoint/globus_compute_endpoint/engines/high_throughput/engine.py

```diff
@@ -0,0 +1,924 @@
+"""HighThroughputExecutor builds on the Swift/T EMEWS architecture to use MPI for fast
+task distribution
+
```

Comment:
There are a number of references to `HighThroughputExecutor` in this file, including in documentation and comments.  Ought these be modified?

Reply:
Fixed.